local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -39,12 +39,13 @@ function launchDevTools({port, projectRoots}, isChromeConnected) {
     });
   } else if (!isChromeConnected()) {
     // Dev tools are not yet open; we need to open a session
-    launchChromeDevTools(port);
+    launchChromeDevTools(host);
   }
 }
 
 module.exports = function(options, isChromeConnected) {
   return function(req, res, next) {
+    var host = req.headers.host;
```

Comment:
Same here.  https://github.com/facebook/react-native/pull/17720/files#r163449140

Reply:
Later it might be nice to honor the protocol too (ex: setting up an HTTPS reverse proxy in front of the packager server) and to use something like Express’s approach to inferring the protocol: https://github.com/expressjs/express/blob/master/lib/request.js#L306